mscore/scoreview.cpp

```diff
@@ -3167,6 +3167,29 @@ void ScoreView::startNoteEntry()
 
       is.setSegment(0);
       Note* note  = 0;
+      if (_score->selection().isNone()) {
+            // select first chordrest in first measure on first staff of current page
+            QPointF center = toLogical(QPoint(width() * 0.5, height() * 0.5));
+            Page* p = point2page(center);
```

Comment:
That will probably fail in Continuous View (LayoutMode::Line). There is a single page in this case. So what about finding the first visible measure in the current view. Or the measure closer from the center of the current view?

Also point2page can be 0 in some extreme cases, we could try in the 4th corners if it fails in the center?